studio/localStores/storageExplorer/StorageExplorerStore.js

```diff
@@ -1126,11 +1130,14 @@ class StorageExplorerStore {
           .concat(pathSegments.slice(columnIndex + 1))
           .join('/')
         return () => {
-          return new Promise(async (resolve, reject) => {
+          return new Promise(async (resolve) => {
             const { error } = await this.supabaseClient.storage
               .from(this.selectedBucket.name)
               .move(fromPath, toPath)
-            if (error) reject()
+              if (error) {
+                hasErrors = true
+                toast.error(`Failed to move ${fromPath} to the new folder`)
```

Comment:
think you're supposed to use ui.setNotification ? https://github.com/supabase/supabase/pull/7348/files

Reply:
having troubling importing this the correct way. @joshenlim lmk if i can pair with you on this

Reply:
no worries this is fine for now - i can help to clean it up in another PR too, this whole file needs some clean up to use the ui store anyways